core/domain/html_validation_service.py

```diff
@@ -957,24 +957,39 @@ def get_filename_with_dimensions(exploration_fs, old_filename):
     Args:
         exploration_fs: GcsFileSystem. File system of the Exploration entity.
         old_filename: str. Name of the file whose dimensions need to be
-            calculated.
+            calculated. The filename is not expected to contain the pattern
+            _height_<number>_width_<number>.
 
     Returns:
         str. The new filename of the image file.
+
+    Raises:
+        Exception: The old_filename provided contains the pattern
```

Comment:
Ditto -- it's more an "ends with" thing, not "contains", right? Same thing below as well.